src/sentry/api/urls.py

```diff
@@ -651,6 +651,9 @@
 from ..monitors.endpoints.organization_monitor_processing_errors_index import (
     OrganizationMonitorProcessingErrorsIndexEndpoint,
 )
+from ..monitors.endpoints.project_processing_errors_details import (
```

Comment:
Heads up @wedamija looks like sometimes your editor is wanting to do parent relative imports

Reply:
It just copies other patterns it sees, so since it sees relative imports it does dumb stuff like this